package.json

```diff
@@ -90,13 +90,16 @@
     "@types/react-dom": "^16.9.10",
     "@types/react-measure": "^2.0.6",
     "@types/react-router-dom": "^5.1.7",
-    "@types/react-slider": "^1.1.0",
+    "@types/react-slider": "^1.1.1",
     "@types/react-window": "^1.8.2",
+    "@typescript-eslint/eslint-plugin": "^4.14.2",
     "babel-loader": "8.1.0",
     "cypress": "^6.3.0",
     "cypress-image-snapshot": "^4.0.1",
-    "eslint": "^7.14.0",
-    "eslint-config-galex": "^2.9.4",
+    "eslint": "^7.19.0",
+    "eslint-config-galex": "^2.10.6",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-testing-library": "^3.10.1",
```

Comment:
So as mentioned, I'm declaring three of the plugins used by `eslint-config-galex` so we can get some of the new rules without waiting for CRA and without having to disable them in our ESLint config.